src/chain.h

```diff
@@ -134,10 +134,18 @@ enum BlockStatus : uint32_t {
     BLOCK_OPT_WITNESS        =   128, //!< block data in blk*.dat was received with a witness-enforcing client
 
     /**
-     * If set, this indicates that the block index entry is assumed-valid.
-     * Certain diagnostics will be skipped in e.g. CheckBlockIndex().
-     * It almost certainly means that the block's full validation is pending
-     * on a background chainstate. See `doc/design/assumeutxo.md`.
+     * If ASSUMED_VALID is set, it means that this block has not been validated
+     * and has validity status less than VALID_SCRIPTS. Also that it has
+     * descendant blocks with VALID_SCRIPTS set, because they were validated
+     * based on an assumeutxo snapshot.
+     *
+     * When an assumeutxo snapshot is loaded, the ASSUMED_VALID flag is added to
+     * unvalidated blocks below the snapshot height. Then, as the background
+     * validation progresses, and these blocks are validated, the ASSUMED_VALID
+     * flags are removed. See `doc/design/assumeutxo.md` for details.
+     *
+     * This flag is only used to implement checks in CheckBlockIndex() and
+     * should not be used elsewhere.
```

Comment:
https://github.com/bitcoin/bitcoin/pull/27746/commits/2d20f75b0bad0b680cd45fc057c54704ac2a9d66

Not entirely true since we have a small usage (by way of `IsAssumedValid()`) in LoadBlockIndex().

Edit: oops nevermind, not as of the fixup commit: https://github.com/bitcoin/bitcoin/pull/27746/commits/78fcc8fbdc510c03dd021e6c34a10e6b35fc2c83